Astronomy/Networking/MarsRoverClient.swift

```diff
@@ -10,14 +10,42 @@ import Foundation
 
 class MarsRoverClient {
     
+    private let baseURL = URL(string: "https://api.nasa.gov/mars-photos/api/v1")!
+    private let apiKey = "qzGsj0zsKk6CA9JZP1UjAbpQHabBfaPg2M5dGMB7"
+    let dataLoader: NetworkDataLoader
```

Comment:
great job making apiKey strongly typed and not using strings which are more error prone.